### PR TITLE
enabling sirius in KCW

### PR DESCRIPTION
**Pull request by gcistaro**
Enabling SIRIUS in KCW for Linear Response

### CODE REVIEW

File: KCW/examples/example01/run_example [hunk 1]
```diff
@@ -3,10 +3,9 @@
 # run from directory where this script is
 cd `dirname $0`
 EXAMPLE_DIR=`pwd`
-
+BIN_DIR="/home/gcistaro/codes/q-e-sirius/spack-build-tklfvgi/bin/"
```
Comment (toxa81) [inline KCW/examples/example01/run_example]: Changes in example01 must be rolled back

File: test-suite/pw_scf/state.h5
Comment (toxa81) [inline test-suite/pw_scf/state.h5:1]: remove h5 file

File: test-suite/pw_scf/scf.in
Comment (toxa81) [inline test-suite/pw_scf/scf.in:1]: undo changes in scf.in

File: test-suite/pw_scf/state.h5
Comment (toxa81) [inline test-suite/pw_scf/state.h5:1]: remove h5 from PR

File: test-suite/pw_scf/scf.in [hunk 2]
```diff
@@ -10,7 +11,7 @@
  &electrons
  /
 ATOMIC_SPECIES
- Si  28.086  Si.pz-vbc.UPF
```
Comment (toxa81) [inline test-suite/pw_scf/scf.in]: roll back to original file, we don't want to touch standard parts of QE

**Review by toxa81 — CHANGES_REQUESTED** [review]
all examples must be returned to QE defaults, h5 files should be deleted from PR

**Comment by toxa81** [comment]
cscs-ci run default

**Comment by toxa81** [comment]
@gcistaro you need to merge the latest `ristretto` branch to your PR to your `interface_kcw` branch to enable ci/cd.

**Comment by gcistaro** [comment]
Thanks Anton, is there a easy way to restore the files to their previous version ?

**Comment by gcistaro** [comment]
Is everything ok now?

**Pull request merged**